### ISSUE TITLE
Error:  ‘cmp’ is not a member of ‘ERT::smspec_node’

### ISSUE THREAD

**Issue by rksin8** [OPM/opm-parser#1125]
Hello, 
        While building opm-parser with dune, I got an error:‘cmp’ is not a member of ‘ERT::smspec_node’. Would you please tell me how to solve this issue? Thank you in anticipation.
![error](https://user-images.githubusercontent.com/22324848/30139211-02b3a5b2-935c-11e7-8d07-183072750364.png)

**Comment by joakim-hove** [comment]
When building the master version of opm-parser you need s quite recent version of libecl, I think that is your problem.

Please install an updated version of  libecl from github.com/Statoil/libecl  - I assume building of opm-parser will work then.

**Comment by rksin8** [comment]
Thank you for your reply!   libecl installation solved the ERT error but another error occurred with opm-core build:
![pppp](https://user-images.githubusercontent.com/22324848/30242362-95db9d8e-9584-11e7-9953-b7fc56a67399.png)

**Comment by atgeirr** [comment]
Looks like some trouble with PETSc, MPI and/or the combination of the two. Since the PETSc code is not used except experimentally it should probably be disabled by default. In your case I am guessing that you have some PETSc installation on your system that is found by the OPM build system, and some insufficiently tested combination of options cause this failure. Do we have an option to explicitly turn off PETSc @jokva or @joakim-hove?

**Comment by joakim-hove** [comment]
> Do we have an option to explicitly turn off PETSc @jokva or @joakim-hove?

Ehhh - apart from the obvious fact that @rksin8 problems are related to MPI in some way - I am *completely at a loss* here - sorry.

**Comment by atgeirr** [comment]
> Ehhh - apart from the obvious fact that @rksin8 problems are related to MPI in some way - I am completely at a loss here - sorry.

Yes, but if it is only triggered by the PETSc-using code it need to be MPI's fault?

**Comment by atgeirr** [comment]
Too late on a Saturday to solve this...

**Comment by rksin8** [comment]
Sorry for the trouble! I removed all old dune installation and did a fresh installation of all (dune+dumux+opm) together. finally, installed successfully!
